script/breakupScript.py

```diff
@@ -0,0 +1,65 @@
+import sys
+
+import numpy
+import numpy as np
+import scipy as sp
+import matplotlib as mpl
+import matplotlib.pyplot as plt
+import pandas as pd
+
+
+def histogram_lc(data, title, cumulative):
+    lc = data["Characteristic Length [m]"]
+    weights = np.ones_like(lc) / float(len(lc))
+    plt.hist(lc, density=False, bins=1000, color='b', weights=weights, cumulative=cumulative)
+    plt.xscale("log")
+    plt.xlabel('$L_c [m]$')
+    plt.ylabel('Relative Probability')
+    if cumulative:
+        plt.title(title + ' $L_c$ Cumulative Distribution')
+    else:
+        plt.title(title + ' $L_c$ Distribution')
+    plt.show()
```

Comment:
If you want you can also directly save the plots with `plt.savefig(filename,dpi=300)` if you want to